src/ViewLayer/DebugDisplay/HomePage/HomePageUi/HomePageUi.cpp

```diff
@@ -1,9 +1,16 @@
 #include "ui_HomePageUi.h"
 #include "HomePageUi.h"
+
 HomePageUi::HomePageUi() :
     ui_(new Ui::HomePageUi)
 {
     ui_->setupUi(this);
+    // https://stackoverflow.com/questions/19939938
```

Comment:
I'd say we don't really need this comment, the rest of this block was used in other pages.